packages/react/src/components/Accordion/Accordion.mdx

```diff
@@ -106,6 +108,19 @@ for layout.
 </Accordion>
 ```
 
+### AccordionItem onHeadingClick
+
+You can use the `onHeadingClick` prop to determine the state of the
+`AccordionItem` when it is clicked. it will return
+
+```jsx
+<Accordion>
+  <AccordionItem onHeadingClick={(evt) => console.log(evt.isOpen)}>
```

Comment:
```suggestion
  <AccordionItem onHeadingClick={({ isOpen }) => console.log(isOpen)}>
```

maybe something like this? or renaming `evt` since it isn't the Event object but is actually an object containing the Event object and the isOpen boolean